src/components/layout/Navbar.tsx

```diff
@@ -0,0 +1,171 @@
+import logo from "/images/logo.jpg";
+import { useState } from "react";
+import {
+  getDefaultEmphasisTextStyles,
+  defaultFlexRowStyles,
+  displayOnMobileStyles,
+} from "../../utils/constants/styles";
+import { useTheme } from "@mui/material/styles";
+import {
+  Box,
+  Typography,
+  SxProps,
+  Button,
+  MenuItem,
+  Menu,
+} from "@mui/material";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faBars } from "@fortawesome/free-solid-svg-icons";
+
+const Navbar = () => {
+  const theme = useTheme();
+  const defaultEmphasisTextStyles = getDefaultEmphasisTextStyles(theme);
+
+  interface LinkProps {
+    name: string;
+    buttonVariant: string;
+    link: string;
+    color?: string;
+  }
+
+  const navbarContainerStyles: SxProps = {
+    ...defaultFlexRowStyles,
+    px: 1.5,
+    py: 1,
+    justifyContent: "space-between",
+    height: "4rem",
+  };
+
+  const imageStyles: SxProps = {
+    height: "100%",
+    objectFit: "contain",
+  };
+
+  const pages: LinkProps[] = [
+    {
+      name: "home",
+      buttonVariant: "text",
+      link: "/",
+    },
+    {
+      name: "about",
+      buttonVariant: "text",
+      link: "/",
+    },
+    {
+      name: "what we do",
+      buttonVariant: "text",
+      link: "/",
+    },
+    {
+      name: "join us",
+      buttonVariant: "contained",
+      link: "/",
+      color: "secondary",
+    },
+  ];
+
+  const [anchorEl, setAnchorEl] = useState<null | HTMLElement>(null);
```

Comment:
For brevity, I think you can just do `useState<HTMLElement?>` instead of an explicit `null | HTMLElement`?

Reply:
Got the feedback-- will fix for next PR.

Reply:
Wait actually this errors when not because we `setAnchorEl(null)` on line 74. This setup was taken directly from the MUI docs.